tools/inundate_nation.py

```diff
@@ -183,32 +191,34 @@ def create_bool_rasters(args):
     profile = raster.profile
     array = raster.read()
     del raster
-    array[array>0] = 1
-    array[array<=0] = 0
+    array[array > 0] = 1
+    array[array <= 0] = 0
     # And then change the band count to 1, set the
     # dtype to uint8, and specify LZW compression.
-    profile.update(driver="GTiff",
-                height=array.shape[1],
-                width=array.shape[2],
-                tiled=True,
-                nodata=0,
-                blockxsize=512, 
-                blockysize=512,
-                dtype='int8',
-                compress='lzw')
-    with rasterio.open(output_bool_dir + os.sep + "bool_" + rasfile, 'w', **profile) as dst:
+    profile.update(
+        driver="GTiff",
+        height=array.shape[1],
+        width=array.shape[2],
+        tiled=True,
+        nodata=0,
+        blockxsize=512,
+        blockysize=512,
+        dtype="int8",
+        compress="lzw",
+    )
+    with rasterio.open(
+        output_bool_dir + os.sep + "bool_" + rasfile, "w", **profile
+    ) as dst:
         dst.write(array.astype(rasterio.int8))
 
 
 def vrt_raster_mosaic(output_bool_dir, output_dir, fim_version_tag):
-    
-  
-    # NOTE: Oct 2022.. we no longer need the VRT, only the large mosaic'd raster. 
+    # NOTE: Oct 2022.. we no longer need the VRT, only the large mosaic'd raster.
     # this code is about to be deprecated, so we will leave it as is.
```

Comment:
As of this PR, this code isn't deprecated. I'd suggest removing this line, potentially the one above as well?